create-rust-app/src/dev/controller.rs

```diff
@@ -37,12 +43,20 @@ pub fn query_db(db: &Database, body: &MySqlQuery) -> Result<String, diesel::resu
 }
 
 /// /db/is-connected
+///
+/// # Panics
+/// * cannot connect to the database
+#[must_use]
 pub fn is_connected(db: &Database) -> bool {
     let mut db = db.pool.clone().get().unwrap();
     let is_connected = sql_query("SELECT 1;").execute(&mut db);
     is_connected.is_err()
 }
```

Comment:
if we fail to connect to the db, then we obviously aren't connected and should return false instead of panicing... this wasn't introduced in this PR so I'll fix it in a new one, but still.

```suggestion
/// /db/is-connected
#[must_use]
pub fn is_connected(db: &Database) -> bool {
    let Ok(mut db) = db.pool.clone().get() else {
        return false;
    };
    let is_connected = sql_query("SELECT 1;").execute(&mut db);
    is_connected.is_err()
}
```

Reply:
fixed in #373